src/envs/preprocess.yaml

```diff
@@ -0,0 +1,10 @@
+name: preprocess
+channels:
+  - conda-forge
+dependencies:
+  - openpyxl
+  - pandas
+  - ca-certificates
+  - openssl
+  - yaml
+  - pyyaml
```

Comment:
Not sure why openpyxl, ca-certificates, openssl, pyyaml are necessary here?

Reply:
Those are just the dependencies it spat out at me that I needed. I started with a fresh environment and only added packages when it asked to try and get the bare minimum. `pyyaml` was needed to get past reading from the `config.yaml` file (not sure how its different from regular `yaml` tbh). Not sure about `ca-certificates` and `openssl`. Does it run without them?

Reply:
Hmm... could you check in the error message to see if it says where the dependencies are needed? They don't seem to be required for me when I remove them and run snakemake all

Reply:
your using the `--use-conda` flag as well, right?